src/components/settings/PushNotifications/PushNotificationsBanner/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { Button, Chip, Grid, SvgIcon, Typography } from '@mui/material'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+import { useCallback, useEffect } from 'react'
+import type { ReactElement } from 'react'
+
+import { CustomTooltip } from '@/components/common/CustomTooltip'
+import { AppRoutes } from '@/config/routes'
+import { useAppSelector } from '@/store'
+import { selectAllAddedSafes, selectTotalAdded } from '@/store/addedSafesSlice'
+import PushNotificationIcon from '@/public/images/notifications/push-notification.svg'
+import useLocalStorage from '@/services/local-storage/useLocalStorage'
+import { useNotificationRegistrations } from '../hooks/useNotificationRegistrations'
+import { transformAddedSafes } from '../GlobalPushNotifications'
+import { PUSH_NOTIFICATION_EVENTS } from '@/services/analytics/events/push-notifications'
+import { trackEvent } from '@/services/analytics'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import CheckWallet from '@/components/common/CheckWallet'
+
+import css from './styles.module.css'
+
+const DISMISS_NOTIFICATION_KEY = 'dismissPushNotifications'
+
+export const PushNotificationsBanner = ({ children }: { children: ReactElement }): ReactElement => {
+  const addedSafes = useAppSelector(selectAllAddedSafes)
+  const totalAddedSafes = useAppSelector(selectTotalAdded)
+  const { safe } = useSafeInfo()
+  const { query } = useRouter()
+
+  const [dismissedBannerPerChain = {}, setDismissedBannerPerChain] = useLocalStorage<{
+    [chainId: string]: boolean
+  }>(DISMISS_NOTIFICATION_KEY)
+
+  const hasAddedSafesOnChain = Object.values(addedSafes[safe.chainId] || {}).length > 0
+  const dismissedBanner = !!dismissedBannerPerChain[safe.chainId]
+
+  const shouldShowBanner = !dismissedBanner && hasAddedSafesOnChain
+
+  const { registerNotifications } = useNotificationRegistrations()
+
+  const dismissBanner = useCallback(() => {
+    trackEvent(PUSH_NOTIFICATION_EVENTS.DISMISS_BANNER)
+
+    setDismissedBannerPerChain({
+      ...dismissedBannerPerChain,
+      [safe.chainId]: true,
+    })
+  }, [dismissedBannerPerChain, safe.chainId, setDismissedBannerPerChain])
+
+  // Click outside to dismiss banner
+  useEffect(() => {
+    if (!shouldShowBanner) {
+      return
+    }
+
+    trackEvent(PUSH_NOTIFICATION_EVENTS.DISPLAY_BANNER)
```

Comment:
Iirc, when I went over the required events with @kirkkonen, this was added to ease funnel creation.

It cannot be moved to `dismissBanner` we show the banner immediately.

Reply:
Maybe to track how many users see the banner but don't dismiss it? But since the banner is closed when clicking anywhere I don't think that would be a useful metric.

Reply:
As per the addition of a close button, this is now not relevant.